maze_transformer/generation/latticemaze.py

```diff
@@ -151,6 +151,33 @@ def as_img(self) -> NDArray["x y", bool]:
 
         return img
 
+    def as_ascii(self, start=None, end=None):
```

Comment:
Regarding the testing of this function - it is indirectly tested by the `test_ascii_encoding`, but that test also tests other logic. Would we want an additional test for this function? If we added one, would we want to mock out this function in the other test? 

I'm never sure about this stuff. @luciaquirke  @valedan do either of you want to chime in?